variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this deployment should be created"
+  default     = "uksouth"
+}
+
+variable "GITHUB_CLIENT_ID" {
+  type = string
+}
+
+variable "GITHUB_CLIENT_SECRET" {
+  type = string
+}
+
+variable "SECRET_KEY" {
+  type = string
+}
```

Comment:
These two should be `sensitive = true` too to make sure Terraform doesn't print them out